examples/add_cdp_asset.ts

```diff
@@ -0,0 +1,31 @@
+import * as BIP39 from "bip39";
+import { CarbonSDK } from "./_sdk";
+import "./_setup";
+import BigNumber from "bignumber.js";
+
+(async () => {
+  const mnemonics = process.env.MNEMONICS ?? BIP39.generateMnemonic();
```

Comment:
unrelated? please verify we're adding this.

Reply:
erm just adding a new example ba, will revert and put in another separate PR